### PR TITLE
Clarify description of graffiti reports

### PR DESCRIPTION
**Pull request by haubourg**

### CODE REVIEW

File: scenarios/html/description.html [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 <p>The aim of this report is to compare QGIS Server performances according to several versions (2.18, 3.4, 3.6 and master branch). Moreover, some configurations are tested too like parallel rendering, trust option, PAL candidates, ... This way, we're able to keep an eye on regressions and improvements!</p>
 
-<p>For now, only <i>GetCapabilities</i> and <i>GetMap</i> requests are tested. Regarding the underlying data, a Postgis database is used (with several kinds of geometries, as described in each tests) as well as GeoTIFF images.</p>
+<p>For now, only <i>GetCapabilities</i> and <i>GetMap</i> requests are tested. Regarding the underlying data, a Postgis database is used (with several kinds of geometries, as described in each tests) as well as GeoTIFF rasters.</p>
+
+<p>The <i>"(Database)"</i> flag in the legend shows the time spent to retrieve data from PostgreSQL as a reference.</p>
```
Comment (pblottiere) [inline scenarios/html/description.html]: `(Database)` -> `(database)`

Comment (haubourg) [reply scenarios/html/description.html]: done @pblottiere

Comment (pblottiere) [reply scenarios/html/description.html]: :+1:

**Comment by pblottiere** [comment]
Thanks @haubourg!

**Pull request merged**